scripts/make_xcframework.sh

```diff
@@ -0,0 +1,122 @@
+clone_and_patch_XNNPACK() {
+    git clone --recurse-submodules https://github.com/google/XNNPACK.git "${XNNPACK_DIR}"
+    cd "${XNNPACK_DIR}"
+    git checkout fb8d1f1b2bb2e32c141564528a39748c4631b453
+    git apply "${PATCHES_DIR}/XNNPACK.patch"
+}
+
+# Bazel does not support arm64 slices for the simulator yet. But there is a workaround
+# See: https://github.com/bazelbuild/rules_apple/issues/980#issuecomment-738645357
+download_patched_version_of_bazel() {
+    mkdir -p "${BAZEL_DIR}/4.1.0"
+    cd "${BAZEL_DIR}/4.1.0"
+    curl -O -L "https://github.com/bazelbuild/bazel/releases/download/4.1.0/bazel-4.1.0-darwin-x86_64"
+    mv bazel-4.1.0-darwin-x86_64 bazel
+    chmod +x bazel
+
+    cd "${BAZEL_DIR}"
+    git clone --branch "3.7.2" --depth 1 https://github.com/bazelbuild/bazel.git "3.7.2"
+    cd "3.7.2"
+    curl -O -L https://gist.githubusercontent.com/indragiek/e14162c0098d97ee976bceae9441f04d/raw/dba3a66a90d0a96d0dc6e236ab3fa1d41a2ce359/ios_sim_arm64.patch
+    git apply ios_sim_arm64.patch
+
+    # To install java11 on macos: brew install java11 https://mkyong.com/java/how-to-install-java-on-mac-osx/
+    # https://github.com/bazelbuild/bazel/issues/11399#issuecomment-628945756
+    # https://github.com/bazelbuild/rules_nodejs/issues/1301
+    export JAVA_HOME="/usr/local/opt/openjdk@11/libexec/openjdk.jdk/Contents/Home" && "${BAZEL_DIR}"/4.1.0/bazel build --incompatible_restrict_string_escapes=false -c opt //src:bazel
+    mkdir -p "$(dirname ${PATCHED_BAZEL_BIN})"
+    cp bazel-bin/src/bazel "${PATCHED_BAZEL_BIN}"
+    chmod +x "${PATCHED_BAZEL_BIN}"
+}
+
+build_xcframework() {
+    readonly SIM_ARM64_DIR="${BUILD_DIR}/iphonesimulator/ios_sim_arm64"
+    readonly SIM_x86_DIR="${BUILD_DIR}/iphonesimulator/ios_x86_64"
+
+    cd ${TENSORFLOW_DIR}
+
+    # See: https://github.com/tensorflow/tensorflow/issues/8527#issuecomment-289272898
+    export TF_CONFIGURE_IOS=1
+    # See: https://github.com/tensorflow/tensorflow/issues/8527#issuecomment-287923871
+    yes '' | ./configure || True
+
+    # Make arm64 simulator slice
+    ${PATCHED_BAZEL_BIN} build --config=ios --cpu=ios_sim_arm64 -c opt tensorflow/lite/ios:${TENSORFLOW_TARGET}
+    mkdir -p "${SIM_ARM64_DIR}" && unzip bazel-bin/tensorflow/lite/ios/${TENSORFLOW_TARGET}.zip -d "${SIM_ARM64_DIR}"
+
+    # Make x86_64 simulator slice
+    bazel build --config=ios --cpu=ios_x86_64 -c opt tensorflow/lite/ios:${TENSORFLOW_TARGET}
+    mkdir -p "${SIM_x86_DIR}" && unzip bazel-bin/tensorflow/lite/ios/${TENSORFLOW_TARGET}.zip -d "${SIM_x86_DIR}"
+
+    # Merge both arm64 and x86_64 simulator slices into a fat framework
+    merge_framework_slices_into_fat_framework \
+        "TensorFlowLiteC" \
+        "${SIM_ARM64_DIR}/TensorFlowLiteC.framework" \
+        "${SIM_x86_DIR}/TensorFlowLiteC.framework" \
+        "${BUILD_DIR}/iphonesimulator"
+
+    # Make ios slices
+    bazel build --config=ios --ios_multi_cpus=armv7,arm64 -c opt tensorflow/lite/ios:${TENSORFLOW_TARGET}
```

Comment:
Is it possible to use patched bazel in all cases.?

Reply:
Discussed offline, and made some changes: added documentation and also use bazelisk instead of bazel